src/assets/i18n/en.json

```diff
@@ -139,8 +139,8 @@
     "agreementEnd": "terms and privacy policy.",
     "removeConsentConfirmation": "Are you sure?",
     "removeConsentConfirmationText": "By doing this the app will stop tracking your location.",
-    "consent": "Consent",
-    "agreementQuestion": "Do you agree with the terms and privacy policy?"
+    "informedConsentTitle": "Provided Information",
+    "informedConsentText": "The Simport Learning App collects your location data and saves it on your device to visualize, aggregate, (identify frequently visited locations) and analyze (estimate work and home location, create predictions) them. If you opt in to have background location on in the App, we will collect data about your location when you are not using the App and when the App is closed (as permitted by your device settings).  <br><br>This data never leaves your local device (i.e., phone or tablet) – we do not upload or save the location data collected to a server. All processing is done locally on the device.  <br><br>The App does not collect any other data."
```

Comment:
I think the text in general reads great, although I think we might need to change / adjust it a bit for the first study:

- This should be removed (because we don't use it): `...and analyze (estimate work and home location, create predictions) them…`
- `This data never leaves your local device (i.e., phone or tablet) – we do not upload or save the location data collected to a server.` - What do you think about adding something like `The option to export and donate your data is completely voluntary.` or saying `This data never leaves your local device (i.e., phone or tablet) unless you export it manually – we do not upload or save the location data collected to a server.` Or is that too much at this point?

Reply:
Overall, I'm okay with these changes. As this PR is pointed to the `develop` branch (from which I'd like to merge it into the `study-1` branch), we can apply your first point of feedback in the `study-1` branch and use the current text in the `develop` branch